source/fundamentals/linq.txt

```diff
@@ -0,0 +1,566 @@
+
+Once the collection is queryable, you can compose a query using 
+**method syntax**. Some pipeline stages also support **query comprehension syntax**,
+which resembles SQL query syntax.
+
+Select the :guilabel:`Method Syntax` or :guilabel:`Query Syntax` tab to see 
+how to compose a query using LINQ:
+
+.. tabs::
+
+   .. tab:: Method Syntax
+      :tabid: method-syntax
+
+      .. code-block:: csharp
+
+         var query = queryableCollection
+             .Where(r => r.Name == "The Movable Feast")
+             .Select(r => new { r.Name, r.Address });
+
+   .. tab:: Query Syntax
+      :tabid: query-syntax
+
+      .. code-block:: csharp
+
+         var query = from r in queryableCollection
+             where r.Name == "The Movable Feast"
+             select new { r.Name, r.Address };
+
+You can print the results of the preceding example as follows: 
+
+.. io-code-block::
+
+   .. input::
+      :language: csharp
+
+      foreach (var restaurant in query)
+      {
+          Console.WriteLine(restaurant.ToJson());
+      }
+
+   .. output::
+      
+      { "name" : "The Movable Feast", "address" : { "building" : "284", "coord" : [-73.982923900000003, 40.6580753], "street" : "Prospect Park West", "zipcode" : "11215" } }
+
+.. tip:: Accessing Query Results
+
+   You can also access the results of your query by using the ``ToList()`` or
+   ``ToCursor()`` methods:
+
+   .. code-block:: csharp
+
+      var results = query.ToList();
+
+   .. code-block:: csharp
+
+      var results = query.ToCursor();
+
+
+Supported Aggregation Stages
+----------------------------
+
+You can use LINQ to create an :ref:`aggregation pipeline <aggregation-pipeline-intro>`.
+The {+driver-short+} automatically translates each LINQ statement into the corresponding
+aggregation pipeline stages. In this section you can learn which
+aggregation pipeline stages are supported.
+
+To learn more about the aggregation pipeline stages, see the 
+:ref:`aggregation-pipeline-operator-reference` page in the server manual.
+
+$project
+~~~~~~~~
+
+The ``$project`` aggregation stage returns a document containing only the specified
+fields.
+
+Select the :guilabel:`Method Syntax` or :guilabel:`Query Syntax` tab to see how 
+to generate a ``$project`` stage using LINQ:
+
+.. tabs::
+
+   .. tab:: Method Syntax
+      :tabid: method-syntax
+
+      .. code-block:: csharp
+         :emphasize-lines: 2
+
+         var query = queryableCollection
+             .Select(r => new { r.Name, r.Address });
+
+   .. tab:: Query Syntax
+      :tabid: query-syntax
+
+      .. code-block:: csharp
+         :emphasize-lines: 2
+
+         var query = from r in queryableCollection
+             select new { r.Name, r.Address };
+
+The result of the preceding example contains the following document:
+
+.. code-block:: json
+
+   { "name" : "The Movable Feast", "address" : { "building" : "284", "coord" : [-73.982923900000003, 40.6580753], "street" : "Prospect Park West", "zipcode" : "11215" } }
```

Comment:
Again: I would have expected `name` to be `Name` and `address` to be `Address`, at least in LINQ3.